assets/css/main.css

```diff
@@ -140,9 +134,9 @@ a:active {
 }
 
 a:focus {
-    border-bottom: 1px solid #0072ce;
-    color: #0072ce;
-    outline: thin dotted;
+    outline: 2px solid {{ site.focus_color }};
```

Comment:
👌

Reply:
Yeah, I think we can change the filename just to `scss` and it would actually still work but for this PR I didn't do anything about it